benchmarks/training_loss.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+import torch
+from datasets import load_dataset
+from datasets import load_metric
+from torch.optim import SGD
+from torch.utils.data import DataLoader
+from transformers import AutoModelForSequenceClassification
+from transformers import AutoTokenizer
+
+import torchdynamo
+
+torchdynamo.config.fake_tensor_propagation = False
+
+# You will download around 84G dataset if you run this end to end training/evaluation example.
```

Comment:
Is there a way to cache this artifact in CI? Or maybe have a flag to run a smaller version?

Reply:
I was thinking to cache it in CI, but it's too big IMO. I have two concerns to have a smaller version:
* We need to generate the smaller version by ourself, so it's not a standard HF training pipeline any more. This is also used as an example/tutorial to help users onboard their models. We are trying to do minimized changes on the way of using HF.
* A smaller version may can't reveal the training loss convergence sometime.

Reply:
Yeah, that seems reasonable, albeit unfortunate. 

cc @malfet for CI caching ideas?